### ISSUE TITLE
CGT layer like tvpaint 

### ISSUE THREAD

**Issue by Milanbkimpe** [tahoma2d/tahoma2d#1380]
It's always long to color animations, that's why I wanted to know if in the futur released, we could see a CGT layer like in tvpaint. A layer with a "grafity" which colors automatically the frames. You won't have to use fill tool for each frame. Thank you.
<img width="1128" alt="Capture d'écran 2024-02-01 203254" src="https://github.com/tahoma2d/tahoma2d/assets/133198562/c898ff2c-7f86-4255-be37-7660636c0363">

**Comment by Codeoloketuyi** [comment]
Do you mean this?

https://github.com/tahoma2d/tahoma2d/assets/137768661/a66b2612-5f55-4984-960c-f09e106f267a

**Comment by Milanbkimpe** [comment]
Yes it’s exactly what I mean. I think it’s really hard to code but that’s it. Thank you si much for your software it’s amazing i love it and the work you do is crazy Thank you. MilanLe 4 avr. 2024 à 00:06, Code Z ***@***.***> a écrit :﻿
Do you mean this?
https://github.com/tahoma2d/tahoma2d/assets/137768661/a66b2612-5f55-4984-960c-f09e106f267a
